dependency_support/load_external.bzl

```diff
@@ -384,6 +384,8 @@ def load_external_repositories():
         strip_prefix = "zstd-1.4.7",
         urls = ["https://github.com/facebook/zstd/releases/download/v1.4.7/zstd-1.4.7.tar.gz"],
         build_file = "//dependency_support/com_github_facebook_zstd:bundled.BUILD.bazel",
+        # Modify decodecorpus to allow generation of zstd frame headers only
```

Comment:
is that a patch worth submitting upstream?

Reply:
I don't think facebook would care about a bazel build :)

Reply:
Ah, you mean the patch. Never mind my comment .